backend/src/email/middlewares/email-transactional.middleware.ts

```diff
@@ -53,6 +57,25 @@ async function sendMessage(
   }
 }
 
+const rateLimit = expressRateLimit({
+  store: new RedisStore({
+    prefix: 'email-transactional:',
+    client: RedisService.rateLimitClient,
+    expiry: 1,
```

Comment:
Perhaps we can extract the window size to config. Makes it easier to test since we can set a longer window time and test if rate limit kicks in.